src/_locales/es.config

```diff
@@ -61,40 +61,40 @@ Valor por defecto
 Fino
 
 @light
-Claro
+ligera
 
 @regular
 Regular
 
 @medium
-Medium
+Medio
 
 @bold
-Bold
+Negrita
 
 @black
-Black
+Negro
 
-@thin_italic
-Thin Italic
+@thin_Italica
+Italica fina
 
-@light_italic
-Light Italic
+@light_Italica
```

Comment:
typo, this needs to remain `@light_italic` (also applies to other @keywords above/below)

Reply:
```suggestion
@light_Italic
```